src/Components/Events/GoogleCalendar/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { MainContainer } from '../style';
+
+function GoogleCalendar({timezone}) {
+  return (
+    <MainContainer>
+        <iframe src={`https://calendar.google.com/calendar/u/0/embed?src=sh10tv3mtfve62somg9nngp9tg@group.calendar.google.com${timezone}`}
+          title="Calendar" style={{border: "0", width: "100%", height:"500px", frameborder:"0", scrolling:"no"}}>
```

Comment:
can we move style from inline?

Reply:
are pixels necessary ?

Reply:
@annabauza  I tried adding react-native StyleSheet for the iframe, but the iframe component can't receive style created by StyleSheet.create(), so I have kept inline style for it. Please let me know if there is any other way to do it.
And pixels are necessary to get the required height.

Reply:
Fair enough